src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -538,6 +538,21 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                                    : args.value;
         },
         true, "a = am/pm, zzz = milliseconds");
+
+    layout.addDropdown<QString>(
```

Comment:
Please move this dropdown to the Moderation sidebar, under the Logs tab, and un-titlecase the label

Reply:
This looks fine to me - it can be updated to match at a later point, or upon request.